android/build.gradle

```diff
@@ -1,13 +1,13 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    ext.kotlin_version = '1.3.72'
+    ext.kotlin_version = '1.5.0'
     repositories {
         google()
         mavenCentral()
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:7.0.2"
+        classpath "com.android.tools.build:gradle:7.1.3"
```

Comment:
I'm a little confused here, is crt using 7.1.2? Would we need to use the same version?

Reply:
Ah, I think I came to this change separately from the CRT. These dependencies only get used during the specific build so I don't think they need to stay the same between CRT and here. IIRC crt was using 7.0.0 and it was 7.0.2 here? Changing it to 7.1.2 for consistency!